saleor/checkout/calculations.py

```diff
@@ -0,0 +1,48 @@
+from typing import TYPE_CHECKING
+
+from ..extensions.manager import get_extensions_manager
+
+if TYPE_CHECKING:
+    from prices import TaxedMoney
+    from .models import Checkout, CheckoutLine
+    from ..discount.types import DiscountsListType
+
+
+def checkout_shipping_price(
+    checkout: "Checkout", discounts: "DiscountsListType" = None
+) -> "TaxedMoney":
+    """Return checkout price without taking in account aby plugins and extensions.
```

Comment:
`aby` -> `any` ?

Reply:
I am not sure. In the first sentence, you are writing `without taking in account aby plugins and extensions` but the second sentence says - `It takes in account all extensions.`. Am I missing something?

Reply:
Typo